pkgs/applications/misc/onagre/default.nix

```
@@ -5,6 +5,8 @@
 , pkgconf
 , freetype
 , expat
+, xorg
```

Do not use xorg indirection.

```suggestion
, libX11
, libXcursor
, libXi
, libXrandr
```